wiki/Tournaments/MWC/2018/7K/en.md

```diff
@@ -0,0 +1,211 @@
+
+| Position | Member |
+| ------------ | -------------- |
+| Management | [![][flag_AR] juankristal](https://osu.ppy.sh/users/443656), [![][flag_DE] p3n](https://osu.ppy.sh/users/123703), [![][flag_ES] Deif](https://osu.ppy.sh/users/318565), [![][flag_US] HappyStick](https://osu.ppy.sh/users/776257), [![][flag_HK] mangomizer](https://osu.ppy.sh/users/1893718), [![][flag_CL] WalterToro](https://osu.ppy.sh/u/5281416) |
+| Map Selectors | [![][flag_ES] aitor98](https://osu.ppy.sh/u/3154852), [![][flag_US] Blocko](https://osu.ppy.sh/u/4075092), [![][flag_BR] spoonguy](https://osu.ppy.sh/u/932381) |
+| Commentators | [![][flag_SE] \[ Vento \]](https://osu.ppy.sh/u/1612580), [![][flag_AU] Lusty Platypus](https://osu.ppy.sh/u/2956184),  [![][flag_AU] PotassiumF](https://osu.ppy.sh/u/4247722), [![][flag_US] TheToaphster](https://osu.ppy.sh/u/7616811), [![][flag_FR] XeoStyle](https://osu.ppy.sh/u/3377280) |
+| Statisticians | [![][flag_NZ] deadbeat](https://osu.ppy.sh/users/128370), [![][flag_DE] Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/687190)
+- [Livestream](https://www.twitch.tv/osulive)
+- [Sign up here!](https://osu.ppy.sh/community/tournaments/13)
+
+------------------------------------------------------------------------
+
+## Participants
+
+The participants will be displayed once the registration phase is over.
+
+------------------------------------------------------------------------
+
+## Mappools
+
+The mappools will be revealed one week in advance of each stage and will be displayed in here whenever that moment takes place.
+
+------------------------------------------------------------------------
+
+## Ruleset
+### Tournament Rules
+
+1. The osu!mania 7K World Cup is a country-based team tournament, played on the osu!mania game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
+
+2. Beatmap scoring is based on Score V2.
+
+3. The beatmaps for each round will be announced by the mapset selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
+   - The Group Stage mappool will be announced after the drawings.
+   - One beatmap will be given as a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players scores do not get added to the team score.
+   - Reviving and surviving during a beatmap gets considered as passing it.
+
+7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
+8. If the beatmap ends in a draw, the game will be nullified.
```

Comment:
I think an explanation on what a game nullification actually means would be preferred